lib/tutorials/en_US/getting-started.md

```diff
@@ -18,23 +19,22 @@ The most basic server looks like the following:
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = new Hapi.Server({ port: 3000, host: 'localhost' });
 
-server.start((err) => {
-
-    if (err) {
-        throw err;
-    }
-    console.log(`Server running at: ${server.info.uri}`);
-});
+server.start()
```

Comment:
the v17 rewrite was to emphasize async/await, perhaps we should be doing the same in our tutorials?

the gotcha there is that you can't use await at the top level of a module, so we would have to

```js
const internals = {};
internals.init = async () => {
  // all the startup code goes here
  // creating the server, await server.start(), etc
};

internals.init();
```

thoughts?

Reply:
Yes, every async method should have an await in examples.